src/main/java/org/example/src/game/Game.java

```diff
@@ -1,68 +1,93 @@
 package org.example.src.game;
 import org.example.src.units.*;
+
+
+import java.io.IOException;
 import java.util.Scanner;
 
 public  class Game {
-    protected static Hero hero ;
+    protected static Hero hero;
     protected static Unit enemy;
 
+    protected static SaveSystem saveSystem;
+
+    static {
+        try {
+            saveSystem = new SaveSystem();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
     protected static Scanner input = new Scanner(System.in);
-    protected static void setHero(String name){
+
+    protected static void setHero(String name) {
         hero = new Hero(name);
     }
-    protected static void setEnemy(Unit obj){
+
+    protected static void setEnemy(Unit obj) {
         enemy = obj;
     }
-    protected static void getAction(){
+
+    protected static void getAction() {
+        boolean save;
         upper:
-        while(true) {
+        while (true) {
+            save = false;
             int choice = input.nextInt();
             try {
                 switch (choice) {
-                    case 1 :{
+                    case 1: {
```

Comment:
Generally, you might like to try using the enhanced switch when dealing with one-liners. Notice how you can move these breaks outside the switch - you wouldn't have to use the upper label to break the loop.